cli/src/main/java/org/infinispan/cli/commands/kubernetes/Uninstall.java

```diff
@@ -1,26 +1,28 @@
 package org.infinispan.cli.commands.kubernetes;
 
-import static org.infinispan.cli.commands.kubernetes.Kube.OPERATOR_SUBSCRIPTION_CRD;
-
-import java.io.IOException;
+import java.util.Map;
 
 import org.aesh.command.CommandDefinition;
 import org.aesh.command.CommandResult;
 import org.aesh.command.option.Option;
 import org.infinispan.cli.commands.CliCommand;
 import org.infinispan.cli.impl.ContextAwareCommandInvocation;
-import org.infinispan.cli.impl.KubernetesContextImpl;
+import org.infinispan.cli.impl.KubernetesContext;
+import org.infinispan.cli.logging.Messages;
+import org.infinispan.commons.util.Version;
 
+import io.fabric8.kubernetes.api.model.GenericKubernetesResource;
 import io.fabric8.kubernetes.client.KubernetesClient;
+import io.fabric8.kubernetes.client.dsl.Resource;
 
 /**
  * @author Tristan Tarrant &lt;tristan@infinispan.org&gt;
  * @since 12.0
  **/
-@CommandDefinition(name = "uninstall", description = "Uninstalls the operator")
+@CommandDefinition(name = "uninstall", description = "Removes the Operator.")
 public class Uninstall extends CliCommand {
 
-   @Option(shortName = 'n', description = "Select the namespace")
+   @Option(shortName = 'n', description = "Specifies the namespace where the Operator is installed, if you did not install it globally.")
```

Comment:
@tristantarrant FYI I just pushed some doc suggestions. There's one change I'm not 100% on here. The namespace arg with uninstall is for Operators scoped to a specific namespace right?

Reply:
Yes